src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php

```diff
@@ -84,8 +84,10 @@ abstract class AbstractLogEntry implements LogEntryInterface
      * @var array<string, mixed>|null
      *
      * @ORM\Column(type="array", nullable=true)
+     *
+     * @note The attribute uses the "array" name directly instead of the constant since it was removed in DBAL 4.0.
```

Comment:
Could we convert this into a `@todo` message explaining how to proceed when possible?